src/documents/ContextAwareness.md

```diff
@@ -0,0 +1,327 @@
+---
+title: ContextAwareness API
+layout: default
+category: API
+---
+
+The ContextAwareness API of Infusion provides a powerful suite of grades and utilities that allow a component to be responsive to aspects of its contexts in a flexible and open-ended way.
+For lightweight cases of context adaptation, the [`distributeOptions`](IoCSS.md) scheme suffices, in that it enables a component to be responsive to broadcasts "advising" it from around the
+component tree in a free-form and controllable away. However, when a component needs to take a clearer role in responding to potentially many kinds of requests for adaptation, where these
+can be organised around two or more "axes" or "dimensions" of adaptation that can be clearly named and identified, it should opt into this more heavywight scheme by implementing the 
```

Comment:
Instead of "it should opt into this more heavyweight scheme by implementing the 'fluid.contextAware' grade" perhaps just say "it should use the 'fluid.contextAware' grade"